app/main.ipynb

```diff
@@ -403,23 +489,65 @@
     "result"
```

Comment:
The `result` variable is not updated with the output of `unknown_signature`. Consider assigning the function's return value to `result`.